### ISSUE TITLE
Change default local site TLD

### ISSUE THREAD

**Issue by philipnewcomer** [philipnewcomer/wpdocker#94]
Since Chrome [is now forcing all `.dev` domains to use HTTPS](https://ma.ttias.be/chrome-force-dev-domains-https-via-preloaded-hsts/), we should change the default TLD for local sites to something else, I propose `.localhost`. However, this new default TLD should be configurable in the Pilothouse config file, as there does not appear to be a consensus in the industry on what the new local host TLD should be, and some folks may still want to use `.dev` by default if HTTPS is not a problem for their local sites.

**Comment by AaronHolbrook** [comment]
Agreed - and it's going to take a while to switch all existing sites/etc and change our frame of mind

**Comment by philipnewcomer** [comment]
I saw in the most recent Docker update release notes that "there is an RFC banning the use of subdomains of localhost". We may want to reconsider the default local site TLD of `.localhost`.

<img width="498" alt="screen shot 2018-01-11 at 9 29 04 am" src="https://user-images.githubusercontent.com/1446874/34830459-e43a5360-f6b1-11e7-99c7-b23fc76fda39.png">

**Comment by philipnewcomer** [comment]
`.localhost` is definitely going to be a problem. Subdomains of `localhost` will always be resolved directly to 127.0.0.1 by the operating system, bypassing any hosts file entries. That would be an advantage on the host, eliminating the need for hosts file entries. However, that would also make it so that PHP scripts would be unable to connect back to themselves through the Nginx container, since the site URL would always resolve to the Docker container the code is running in, and the PHP containers do not accept HTTP connections. A PHP script could never make a connection to `sitename.localhost`. That would, for instance, [break WP cron](https://github.com/Pilothouse-App/Pilothouse/issues/86).

**Comment by Jaace** [comment]
I have started using `.test` for my sites.

**Comment by AaronHolbrook** [comment]
I dislike that name myself as it is not representative of what I’m using the site for.

I.e. I’m not testing something, I’m developing something - I still prefer .dev if I can help it.

Whatever the case is, I think making this modifiable by the user makes sense so we can allow people to adjust this should there be any issues down the road.
